spec/services/open_library_service_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe OpenLibraryService do
+  response_path = './spec/fixtures/stubs_and_mocks/open_library_service_response_body.txt'
```

Comment:
[Check this](https://relishapp.com/rspec/rspec-rails/docs/file-fixture) rspec helper about fixtures!